docs/www/shadow-indexing.md

```diff
@@ -0,0 +1,401 @@
+---
+title: Data Management
+order: 1
+---
+
+# Data Management 
+
+One of the key aspects of working with Kafka is how you manage your data.
+
+Creating and maintaining backup scripts is always a pain in the neck and although it can improve reliability it doesn't bring value to the customer directly.
+
+This is the type of work that we would like to avoid and Redpanda can help you with that.
+
+Introducing **Shadow Indexing**.
+
+## Shadow Indexing
+
+**Shadow Indexing** is a unique capability in Redpanda that allows you to archive your data to object storage and recreate complete indexes from it, for all your historic data. 
+
+This capability unlocks new features such as replicating topics from one cluster to another, disaster recovery, etc. without burdening your clusters with local storage and compute.
+
+It's all implemented with [minio](https://min.io/) so it should work with any cloud provider. We officially support Amazon's S3 and Google Cloud Storage. 
+
+You just have to configure your cloud credentials and we'll do the hard work for you. 
+
+With **Shadow Indexing**, Redpanda eliminates the need for disaster recovery by integrating with these infinitely scalable storage systems. It will push the old segments onto the cloud and fetch them when clients need them, with almost no risk to data loss.  
```

Comment:
```suggestion
With **Shadow Indexing**, Redpanda facilitates your disaster recovery plan by integrating with these infinitely scalable storage systems. It will push the your segments as fast as it can onto the cloud and fetch them when clients need it, with almost no risk to data loss.  
```

Reply:
wdyt?

Reply:
lgtm